app/javascript/controllers/polling_controller.js

```diff
@@ -1,15 +1,15 @@
 import { Controller } from "@hotwired/stimulus"
 
 const DEFAULT_DEBOUNCE = 1000
+const DEFAULT_MAX = 120
```

Comment:
Added a new setting to the polling controller, so it'll stop after 2 minutes.

Combined with the `upload_stalled?` helper method, we can mark these unresponsive files as "errored", and let you retry processing them.